p2p/pre2p/telemetry/metrics.go

```diff
@@ -0,0 +1,20 @@
+package pre2p_telemetry
+
+const (
+	// How to define metrics in this file?
+	//
+	// Must define metrics using following template: MetricName_MetricType_Attribute
+	// e.g: type= timeseries, name=nodes_alive_counter, attribute=name
+	// will result in: NODES_COUNT_COUNTER_TIME_SERIES_NAME="nodes_alive_counter"
+	// you can also add: NODES_COUNT_COUNTER_TIME_SERIES_DESCRIPTION="for your description"
+
+	// Time Series Metrics
+	P2P_NODE_STARTED_TIMESERIES_METRIC_NAME        = "p2p_nodes_started_counter"
```

Comment:
Do we need `METRIC` in here or do you think it's redundant

Reply:
I think it's semantic and helps with clarity although verbose, let's keep it as is.